pypeit/data/__init__.py

```diff
@@ -0,0 +1,2 @@
+
+from .utils import *
```

Comment:
My understanding is that this kind of import should be avoided.  Because of that I've always avoided it and don't have much experience with appropriate usage.  It might be okay to leave it in as long as you define `__all__` in `utils.py`.  For example, see [here](https://docs.python.org/3/tutorial/modules.html#importing-from-a-package) .  Maybe others in the team will have some better advice.

Reply:
`__init__.py` is actually one of the few instances where this is the kosher thing to do. you avoid `import *` in most cases to avoid namespace pollution/conflicts. in this case, though, you're collecting multiple namespaces into one. this lets the code be modularized while keeping imports more concise. see `photutils` as an example: https://github.com/astropy/photutils/blob/main/photutils/__init__.py

Reply:
note the use of `# noqa` there. most code checkers/linters will whinge about `import *` and that tells them to back off. maybe add that here as well as future-proofing against stricter code-style tests...

Reply:
Added `# noqa` (`__init__.py`) and `__all__` (`utils.py`) to control the flow of the `import *`.